cli/azd/pkg/input/console.go

```diff
@@ -204,6 +207,16 @@ func (c *AskerConsole) spinnerText(title, charset string) string {
 	return title
 }
 
+func (c *AskerConsole) Progress(ctx context.Context, progress string) {
+	if c.spinner == nil {
+		return
+	}
+
+	_ = c.spinner.Pause()
+	c.spinner.Message(fmt.Sprintf("%s (%s)", c.currentSpinnerMessage, progress))
+	_ = c.spinner.Unpause()
```

Comment:
Do we need to pause and unpause? My understanding is that `Message` is atomic in regards to the title update. This is only necessary if you're also changing other properties.